README.md

```diff
@@ -106,6 +106,22 @@ cmake --build . --parallel -t vgg_container
 cmake --install . --prefix <path/to/vgg_ios/VggRuntime/external>
 ```
 
+#### Android Building example
+
+We currently only support Android NDK = 27 and Android API >= 24.
+
+You need to download Android NDK r27 from here: https://github.com/android/ndk/releases/tag/r27
+
+Build & install `vgg_container` libraries for [vgg_android](https://github.com/verygoodgraphics/vgg_android).
+
+```bash
+mkdir build.android
+cd build.android
+cmake .. -DCMAKE_TOOLCHAIN_FILE=<path_to_android_ndk>/android-ndk-r27/build/cmake/android.toolchain.cmake -DVGG_VAR_TARGET="Android-x86_64" -DANDROID_NDK=<path_to_android_ndk>/android-ndk-r27
```

Comment:
Maybe the explicit directory name `android-ndk-r27` could be removed? We could assume `<path_to_android_ndk>` already includes it.